feature/amazon/src/main/AndroidManifest.xml

```diff
@@ -3,13 +3,19 @@
         package="com.revenuecat.purchases.amazon">
 
     <application>
-        <receiver android:name = "com.amazon.device.iap.ResponseReceiver"
-                android:permission = "com.amazon.inapp.purchasing.Permission.NOTIFY"
-                android:exported = "true">
+        <receiver
+                android:name="com.amazon.device.iap.ResponseReceiver"
+                android:exported="true"
+                android:permission="com.amazon.inapp.purchasing.Permission.NOTIFY">
             <intent-filter>
-                <action android:name = "com.amazon.inapp.purchasing.NOTIFY" />
+                <action android:name="com.amazon.inapp.purchasing.NOTIFY" />
             </intent-filter>
         </receiver>
+        <activity
+                android:name=".ProxyAmazonBillingActivity"
+                android:configChanges="keyboard|keyboardHidden|screenLayout|screenSize|orientation"
+                android:exported="false"
+                android:theme="@android:style/Theme.Translucent.NoTitleBar" />
```

Comment:
It's redundant but I added it as a backup in case `setTheme` in the activity doesn't work. Looking at all the problems we had setting the theme the other day I wouldn't be surprised `setTheme` doesn't work. What do you think? I am happy to remove it too

Reply:
Well, I think it would be confusing to have it in both places... Unless we know this is needed I think I would vote to remove it. Not a strong opinion though.